Dockerfile-worker

```diff
@@ -11,12 +11,16 @@ ENV CELERYD_TIME_LIMIT=36000
 WORKDIR /usr/local/app
 
 # add path where simulator binaries will be installed
-ENV PATH="/usr/local/app/bin:${PATH}"
+ENV PATH="/simulators/shieldhit12a/bin:${PATH}"
+ENV PATH="/simulators/topas/bin:${PATH}"
 
 # openshift permissions
 RUN chgrp -R 0 /usr/local/app && \
     chmod -R g=u /usr/local/app
 
+# set variable with path to Geant4 files required by TOPAS simulator
+ENV TOPAS_G4_DATA_DIR="/simulators/geant4"
+
```

Comment:
can you move that above openshift stuff ? we should gather simulator related lines close to each other